src/Cake.Common/Tools/NUnit/NUnit3Labels.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.Common.Tools.NUnit
+{
+    /// <summary>
+    /// Represents the possible values for the Labels option
+    /// </summary>
+    public enum NUnit3Labels
+    {
+        /// <summary>
+        /// Does not output labels, the default
```

Comment:
Should probably be something like `Does not output labels. This is the default.` right?